examples/flutter_gallery/lib/welcome/home.dart

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Overall comment: I could not find a way of running the onboarding experience again after the first run of the app. I suggest we add a new entry 'Onboarding experience' as the last item in the list under 'Studies' that relaunches it.

Reply:
Added